README.md

```diff
@@ -2094,6 +3716,19 @@ Searches the given object by the given predicate and returns the found entry or
 ```
 <!-- prettier-ignore-end -->
 
+##### Examples
+
+<!-- prettier-ignore-start -->
+```javascript
+findEntry(({ x }) => x % 2 === 0)({ a: { x: 1 }, b: { x: 2 }, c: { x: 3 } });
+// ⇒ ["b", { x: 2 }]
+```
+<!-- prettier-ignore-end -->
+
+##### Questions
+
+- How to find an entry of an object by a function?
```

Comment:
`by` => `with` ?

Reply:
I have clarified that it is about a `predicate function` and kept the `by` form.